src/cdk/testing/protractor/protractor-element.ts

```diff
@@ -212,5 +212,5 @@ export class ProtractorElement implements TestElement {
 function _dispatchEvent(name: string, element: ElementFinder) {
   const event = document.createEvent('Event');
   event.initEvent(name);
-  element.dispatchEvent(event);
+  element['dispatchEvent'](event);
```

Comment:
Maybe we should have a comment on why it needs to be written this way?

Reply:
Done 👍🏽